### DISCUSSION TITLE
请教CefShutdown的问题

### DISCUSSION THREAD

**Discussion by ajian55** [CefView/QCefView#144] [General]
请教一下，在程序退出前调用CefShutdown，耗时很长，要好几秒钟。导致以下现象：退出时，程序界面首先关闭，然后又迅速自动打开界面，直到CefShutdown调用结束，又自动关闭界面，此时整个程序才真正退出。请问如何避免程序关闭后界面又自动打开问题？

**Comment by tishion** [comment]
> 请教一下，在程序退出前调用CefShutdown，耗时很长，要好几秒钟。导致以下现象：退出时，程序界面首先关闭，然后又迅速自动打开界面，直到CefShutdown调用结束，又自动关闭界面，此时整个程序才真正退出。请问如何避免程序关闭后界面又自动打开问题？

是你自己调用CefShutdown？

**Reply by ajian55** [reply]
是的，受限于一些因素，暂时只能使用VS2015及较低版本的Qt，QCefView没法直接用。因此想试试直接用Cef，需求简单，只需要内嵌打开网页（QWebEngine也用不了）。但是仍然遇到了上述问题。求指教。

**Reply by tishion** [reply]
代码里面都写了，这个问题比较难解释，你配合代码和cef的文档理解吧。

**Comment by ajian55** [comment]
收到。谢谢！
